app/controllers/recipes_controller.rb

```diff
@@ -1,3 +1,41 @@
 class RecipesController < ApplicationController
-  def index; end
+  before_action :authenticate_user!
+
+  def index
+    @recipes = Recipe.all
```

Comment:
I think we should retrieve only the recipes of the current_user in the index of this controller. That way we won't be able to see recipes from other users in our own list of recipes.

We will have a public recipes list later on.

Reply:
You're right! 👍